build-parent/pom.xml

```diff
@@ -35,7 +35,12 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jms-spi-deployment</artifactId>
+                <version>${quarkus.version}</version>
```

Comment:
Do we already have a PR in quarkus to fix this? If so: Can we add a TODO to remove this entry?

Reply:
Relevant PR for `2.16`: https://github.com/quarkusio/quarkus/pull/31075

Reply:
@zhfeng can you add the `TODO` in the pom? As far as I am concerned, this is the last blocker on the PR. I'd like to get this one merged before we switch the main to `3.0.0`-based implementation.

Reply:
Yeah, I can add it.